release_process.md

```diff
@@ -24,20 +24,23 @@ indexes.
 The release process uses a handful of GitHub actions to automate the process.
 
 ```text
-  ┌─────────────────────┐                                                
-  │Create Release Commit│                                                
-  └─┬───────────────────┘                                                
-    │                           ┌────────────┐ ┌──►Python GH Release     
-    ├──►(tag) python-vX.Y.Z ───►│PyPI Publish├─┤                         
-    │                           └────────────┘ └──►Python Wheels         
-    │                                                                    
-    │                           ┌───────────┐                            
-    └──►(tag) vX.Y.Z ───┬──────►│NPM Publish├──┬──►Rust/Node GH Release  
-                        │       └───────────┘  │                         
-                        │                      └──►NPM Packages          
-                        │       ┌─────────────┐                          
-                        └──────►│Cargo Publish├───►Cargo Release         
-                                └─────────────┘                          
+  ┌─────────────────────┐
+  │Create Release Commit│
+  └─┬───────────────────┘
+    │                           ┌────────────┐ ┌──►Python GH Release
+    ├──►(tag) python-vX.Y.Z ───►│PyPI Publish├─┤
+    │                           └────────────┘ └──►Python Wheels
+    │
+    │                           ┌───────────┐
+    └──►(tag) vX.Y.Z ───┬──────►│NPM Publish├──┬──►Rust/Node GH Release
```

Comment:
Just a note for us in the future: we should probably add `java` to these release notes. And also consider taking seriously the language labels in the release notes. Think about what we need to do to make them easy to read for both Node and Java developers.